### ISSUE TITLE
renderer. line in Demo&Tutorial3-8.cs Deprecated

### ISSUE THREAD

**Issue by ryanshackelford** [ricardojmendez/LibNoiseTutorials#2]
renderer.material...

to

GetComponent<MeshRenderer>().material....

```
Demo.cs
line 127
GetComponent<MeshRenderer>().material.mainTexture = this.m_textures [0];

Tutorial3.cs
line 35
GetComponent<MeshRenderer>().material.mainTexture = image;

Tutorial4.cs
line 44
GetComponent<MeshRenderer>().material.mainTexture = image;

Tutorial5.cs
line 66
GetComponent<MeshRenderer>().material.mainTexture = image;

Tutorial6.cs
line 56
GetComponent<MeshRenderer>().material.mainTexture = image;

Tutorial7.cs
line 67
GetComponent<MeshRenderer>().material.mainTexture = image;

Tutorial8.cs
line 26
GetComponent<MeshRenderer>().material.mainTexture = image;

**Comment by DerkJanSpeelman** [comment]
Confirmed. I have the same error

**Comment by ricardojmendez** [comment]
Should be fixed by #4. Please re-open if it persists, I'm not actively using LibNoise.
